Makefile

```diff
@@ -53,7 +53,7 @@ image:
 # Generate the protobuf python files
 proto:
 	shopt -s globstar
-	python3.6 -m grpc.tools.protoc \
+	python3 -m grpc.tools.protoc \
```

Comment:
what is recommended python version currently? I think 3.6+ works fine?

Reply:
On my machine, I have python 3.7+ installed.
That's why I simply used `python3` for this command as I don't have python3.6 installed.
However we should specify the fabric-sdk-py code only run with `python3.6+` interpreters ;)